servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Publisher.java

```diff
@@ -188,6 +188,156 @@ protected Publisher() {
         return new ResumePublisher<>(this, nextFactory, executor);
     }
 
+    /**
+     * Each element of this {@link Publisher} is mapped into a {@link Publisher} (potentially of a different type) and
```

Comment:
I will make them consistent. I prefer using the operator method name in the description (e.g. `flatten` and `map`) to make the method/operator naming more clear (otherwise `flatMap` can be a bit esoteric if you are not familiar with these functional concepts).